README.md

```diff
@@ -15,5 +15,23 @@ included.
 You can run `must-gather` on a cluster with `oc adm must-gather`. 
 Use `-h` flag to see available options.
 
+## How to test
+Build a new must-gather image with your changes using the Makefile. There are two options in the Makefile, `make` will use imagebuilder to build the image while `make image-with-podman` will use podman to build the image.
```

Comment:
This line is no longer valid. There's no `make image-with-podman` anymore.

Reply:
Updated docs based on new instructions